source/systems/gaea_user_guide.rst

```diff
@@ -1290,6 +1345,210 @@ For SHMEM codes, users must load the xt-shmem module before compiling:
 
   $ module load xt-shmem
 
+Debugging with DDT
```

Comment:
There's also a DDT section on the Hera page.  Should we break these out and put them in a dedicated DDT page under "software"?  (Or are the implementations completely different?)

Reply:
Until the DDT documentation is updated (I'm almost done with my modifications), we should leave these here.  I'll adjust the DDT sections as the last part of the DDT PR.

Reply:
I think we leave it in two places for now. We're planning to sort of combine the Hera and Jet documentation going forward. And the DDT discussion in the Gaea page is new, so the Hera pages may need a rewrite or redirect when we do that.